src/ui/public/courier/fetch/request/segmented.js

```diff
@@ -33,9 +34,13 @@ define(function (require) {
      *********/
 
     SegmentedReq.prototype.start = function () {
+      var self = this;
+
       this._complete = [];
       this._active = null;
       this._segments = [];
```

Comment:
I know it's a bit redundant but it feels complete to set `this._queueCreated = false;` here.